source/quickguides/generate_quick_guides.py

```diff
@@ -31,14 +39,17 @@
 
 # Important: Do not have more than one `#expand` operation per line. The code will only expand the first one.
 def generate_quick_guides():
-    machine_names = ['acme1', 'compy', 'cori']
-    git_top_level = subprocess.check_output('git rev-parse --show-toplevel'.split()).strip().decode('utf-8')
+    machine_names = EXPANSIONS.keys()
+    git_top_level = subprocess.check_output(
+        'git rev-parse --show-toplevel'.split()).strip().decode('utf-8')
```

Comment:
My editor wraps code at 120 characters. I see you ran `autopep8`. Doesn't really matter though as long as the pull request isn't overwhelmed by new-line changes. We probably should try to find a consistent formatting and stick to it though.

Reply:
I usually use `Black` for Python code style and formatting, which isn't configurable and focuses on consistency and following PEP rules. I used `autopep8` in this case because I didn't want to change the single quotes to double quotes.

If necessary, we can configure a standard tool to style and format the codebase.